wurst/math/Raycast.wurst

```diff
@@ -0,0 +1,483 @@
+package Raycast
+import NoWurst
+import Vectors
+import Wurstunit
+
+/* The package defines such geometry as 'ray' for 2D and 3D separately.
```

Comment:
You should add a small code snippet of one of the raycasts in practical use, showing how to handle the returned tuple correctly.